plugins/code-versions/package.json

```diff
@@ -29,6 +29,7 @@
         "@testing-library/jest-dom": "^6.6.3",
         "@testing-library/react": "^16.3.0",
         "@testing-library/user-event": "^14.6.1",
+        "@types/prismjs": "^1",
```

Comment:
Nit: Exact version?

Reply:
it's `1.26.5` for `@types/prismjs` but `"prismjs": "^1.30.0",`, to avoid confusion, I picked just `^1`.